### PR TITLE
Surface GitHub collaborator seat error

### PR DESCRIPTION
**Pull request by salbertson**

### CODE REVIEW

File: lib/error_message_translation.rb [hunk 1]
```diff
@@ -7,6 +7,8 @@ def self.from_error_response(error)
 
     if matches.present? && matches.captures[0] == FORBIDDEN_ERROR_CODE
       matches.captures[1]
+    elsif error.message.include? "You must purchase at least one more seat"
+      "You need to add at least one more seat to GitHub before enabling Hound. https://help.github.com/articles/adding-seats-to-your-organization"
```
Comment (gylaz) [inline lib/error_message_translation.rb]: Maybe shorten this to just `Please add a seat to GitHub to enable Hound. [link]`

File: spec/lib/error_message_translation_spec.rb [hunk 2]
```diff
@@ -45,4 +57,14 @@ def octokit_403_error_message
   def octokit_400_error_message
     "PUT https://api.github.com/teams/3675/memberships/houndci: 400 - Problems parsing JSON. // See: https://developer.github.com/v3"
   end
+
+  def octokit_422_error_message
+<<-ERROR
```
Comment (gylaz) [inline spec/lib/error_message_translation_spec.rb]: How about using `<<~` heredoc, which allows you to indent everything properly?

File: lib/error_message_translation.rb (hunk 1 above)
Comment (gylaz) [reply lib/error_message_translation.rb]: Also maybe use a hereblock.

**Comment by salbertson** [comment]
@gylaz I went through and addressed the style comments. This should be ready now.

File: spec/lib/error_message_translation_spec.rb [hunk 3]
```diff
@@ -34,6 +34,18 @@
...
       end
     end
+
+    context "when error related to adding more GitHub seats" do
+      it "returns a helpful message" do
+        error = double("error", message: octokit_422_error_message)
+
+        result = ErrorMessageTranslation.from_error_response(error)
+
+        expect(result).to(
```
Comment (gylaz) [inline spec/lib/error_message_translation_spec.rb]: Have you been doing this recently? I haven't seen much usage of `.to(` in RSpec. Typically I see this type of line splitting:
```rb
expect(result).
  to eq(...)
```
Thoughts?

File: spec/lib/error_message_translation_spec.rb [hunk 4]
```diff
@@ -34,6 +34,18 @@
...
+          eq("Please add a GitHub seat to enable Hound. https://help.github.com/articles/adding-seats-to-your-organization"),
```
[the remark this answers is not in the document]
Comment (gylaz) [reply spec/lib/error_message_translation_spec.rb]: Want to break this line up to appease Hound?
